### PR TITLE
Console Content

### PR DESCRIPTION
**Pull request by kent-williams**
This PR adds all existing Console content from developer.helium with some updates to images and organization.

### CODE REVIEW

File: docs/use-the-network/console/adding-devices.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+<img
+  src={useBaseUrl("img/use-the-network/console/console-device-details.png")}
+/>
+
+## Important Information When Adding Devices
+
+### Is The Helium Network a Public or Private LoRaWAN?
+
+The Helium Network is a Public LoRaWAN Network. You can read more about the
+unique, blockchain-driven architecture here.
```
Comment (PharkMillups) [inline docs/use-the-network/console/adding-devices.mdx:42]: "here" missing link

File: docs/use-the-network/console/migrating-devices/migrating-devices.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+The following is a list of network-specific migration guides. Join us!
+
+### Migrating from The Things Network to Helium
+
+- [Importing TTN Devices with Console Tool](/docs/use-the-network/console/migrating-devices/ttn-import)
+- [Manually Migrating Devices from TTN](/docs/use-the-network/console/migrating-devices/ttn-manual)
+
+### Migrating from Senet to Helium \(**Coming Soon**\)
+
+### Migrating from MachineQ to Helium \(**Coming Soon**\)
```
Comment (PharkMillups) [inline docs/use-the-network/console/migrating-devices/migrating-devices.mdx]: Pull both the MQ and Senet links. We won't have them for a while.

File: docs/use-the-network/console/labels.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+Labels provide flexibility to define where to send data from devices. Users can
+apply a Label to an Integration and that Label can be used to connect a single
+or multiple devices to that Integration. For example, users could define an
+internal server as an Integration for initial testing and apply a descriptive
+Label \(e.g., internal_test\). After ensuring devices send data, that
+internal_test Label could easily be replaced by one that maps to a production
+endpoint \(AWS\).
+
+Labels need to be created before attaching them to devices and integrations,
+read more below on how to do both.
```
Comment (PharkMillups) [inline docs/use-the-network/console/labels.mdx]: You can probably kill this entire sentence.

File: docs/use-the-network/console/functions.mdx [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+to store numeric values. For every transmission, we want to pack as much
+information into as few bytes as possible. Because of this, there are special
+encoding standards specifically made for sending common IoT device data such as
+[CayenneLPP](https://developers.mydevices.com/cayenne/docs/lora/#lora-cayenne-low-power-payload).
+
+These types of standards allow you to easily encode your data much more
+efficiently. Once your data has been received over the air by the network, there
+is no longer the necessity of encoding the data with a standard that can make it
+more challenging for humans to interpret. This is where Function Decoders come
+in, they allow you to decode your device data so that you can interpret it for
```
Comment (PharkMillups) [inline docs/use-the-network/console/functions.mdx]: Period after "in". New sentence starting with "They

File: docs/use-the-network/console/integrations/mqtt.mdx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+same label to both the device and integration. Labels need to be created before
+attaching them to devices and integrations. Instructions for doing this are
+[here](../labels).
+
+## Send and Receive Device Data
+
+The following examples are done with the
+[MQTT CLI](https://hivemq.github.io/mqtt-cli) utility.
+
+Example MQTT
```
Comment (PharkMillups) [inline docs/use-the-network/console/integrations/mqtt.mdx:68]: Not sure this is needed.

**Review by PharkMillups — DISMISSED** [review]
left a few small comments. Looks great otherwise.

File: docs/use-the-network/console/adding-devices.mdx (hunk 1 above)
Comment (kent-williams) [reply docs/use-the-network/console/adding-devices.mdx:42]: The relevant content doesn't exist yet I don't think, have been leaving some of these blank until the content appears

File: docs/use-the-network/console/integrations/mqtt.mdx (hunk 5 above)
Comment (kent-williams) [reply docs/use-the-network/console/integrations/mqtt.mdx:68]: The link to the CLI tool or examples all together?

**Review by lthiery — APPROVED** [review]
Progress :rocket:

**Pull request merged**